### PR TITLE
Fix docs checks

### PR DESCRIPTION
**Pull request by tarek-y-ismail**
Adds new words to the custom wordlist to fix the spelling check, and fixes (most) URLs .

### CODE REVIEW

File: doc/sphinx/how-to/getting_involved_in_mir.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 ## Getting involved
 
 The Mir project website is <https://mir-server.io/>,
-the code is [hosted on GitHub](https://github.com/canonical)
+the code is [hosted on GitHub](https://github.com/canonical/mir)
 
 For announcements and other discussions on Mir see:
-[Mir on community.ubuntu](https://community.ubuntu.com/c/mir)
+[Mir on community.ubuntu](https://discourse.ubuntu.com/c/mir/15)
```
Comment (AlanGriffiths) [inline doc/sphinx/how-to/getting_involved_in_mir.md:9]:
```suggestion
[Mir on discourse.ubuntu](https://discourse.ubuntu.com/c/mir/15)
```

File: doc/sphinx/reference/dso_versioning_guide.md [hunk 2]
```diff
@@ -184,8 +184,8 @@ Additionally, there's the complication of passing objects between different
 versions. For the moment, we can not bother trying to make this work.
 
 
-See also: 
+See also:
 ---------
 [Binutils manual](https://sourceware.org/binutils/docs/ld/VERSION.html)
 
-[Former glibc maintainer's DSO guide](http://www.akkadia.org/drepper/dsohowto.pdf)
```
Comment (AlanGriffiths) [inline doc/sphinx/reference/dso_versioning_guide.md:191]: I don't really care, but is there a reason for preferring https for assets that don't benefit from encryption?

**Review by AlanGriffiths — DISMISSED** [review]
A couple of nits

File: doc/sphinx/reference/dso_versioning_guide.md (hunk 2 above)
Comment (tarek-y-ismail) [reply doc/sphinx/reference/dso_versioning_guide.md:191]: The link check doesn't like it :)

Comment (Saviq) [reply doc/sphinx/reference/dso_versioning_guide.md:191]: > I don't really care, but is there a reason for preferring https for assets that don't benefit from encryption?

Privacy.

File: doc/sphinx/reference/continuous-integration.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ available in the `edge` or `beta` channels, as appropriate.
 When the snaps get published to the store, we have Jenkins dispatch test runs on a selection of
 hardware, ranging from Raspberry Pis through single-GPU systems all the way up to high performance
 multi-GPU (usually hybrid) ones. We maintain a matrix of test coverage
-[in this spreadsheet](https://docs.google.com/spreadsheets/d/1kUbTSt4zWVpTtgZNJvvxCdugsRUv6C5PK9Xw5dxppCc/edit#gid=893560997).
+[in this spreadsheet](https://docs.google.com/spreadsheets/d/1kUbTSt4zWVpTtgZNJvvxCdugsRUv6C5PK9Xw5dxppCc/edit?usp=sharing).
```
Comment (Saviq) [inline doc/sphinx/reference/continuous-integration.md:130]:
```suggestion
[in this spreadsheet](https://docs.google.com/spreadsheets/d/1kUbTSt4zWVpTtgZNJvvxCdugsRUv6C5PK9Xw5dxppCc/edit#gid=893560997).
```

This is wrong, the anchor is needed so it links to the right sheet.

**Review by Saviq — APPROVED** [review]
Just one nit.

**Review by Saviq — APPROVED** [review]
Green!

Thanks :)

**Pull request merged**